packages/web/src/scss/themes/index.scss

```diff
@@ -0,0 +1,25 @@
+@use 'sass:map';
+@use 'sass:string';
+@use '@themes' as themes;
+@use '../tools/themes' as themes-tools;
+
+$is-default-theme: true;
+$global-prefix: 'spirit-';
```

Comment:
Somewhere the prefix is configurable, somewhere it's hard-coded.

How about moving it to `scss/settings/_globals.scss` and using it everywhere we use the prefix?